diff --git a/config.yaml b/config.yaml
index 6d651ed..492937f 100644
--- a/config.yaml
+++ b/config.yaml
@@ -1,6 +1,6 @@
 main:
   #components_repository: "https://github.com/udacity/Project-Build-an-ML-Pipeline-Starter.git#components"
-  components_repository: "https://github.com/TexFly/Project-Build-an-ML-Pipeline-Starter"
+  components_repository: "https://github.com/TexFly/Project-Build-an-ML-Pipeline-Starter.git#components"
   # All the intermediate files will be copied to this directory at the end of the run.
   # Set this to null if you are running in prod
   project_name: nyc_airbnb
diff --git a/src/eda/wandb/debug-internal.log b/src/eda/wandb/debug-internal.log
index ff22d79..a013232 120000
--- a/src/eda/wandb/debug-internal.log
+++ b/src/eda/wandb/debug-internal.log
@@ -1 +1 @@
-run-20240208_210022-nvg2zzo1/logs/debug-internal.log
\ No newline at end of file
+run-20240211_102016-lsbwqn83/logs/debug-internal.log
\ No newline at end of file
diff --git a/src/eda/wandb/debug.log b/src/eda/wandb/debug.log
index 2e4d022..5b1f4a6 120000
--- a/src/eda/wandb/debug.log
+++ b/src/eda/wandb/debug.log
@@ -1 +1 @@
-run-20240208_210022-nvg2zzo1/logs/debug.log
\ No newline at end of file
+run-20240211_102016-lsbwqn83/logs/debug.log
\ No newline at end of file
diff --git a/src/eda/wandb/latest-run b/src/eda/wandb/latest-run
index e0f3449..82724a3 120000
--- a/src/eda/wandb/latest-run
+++ b/src/eda/wandb/latest-run
@@ -1 +1 @@
-run-20240208_210022-nvg2zzo1
\ No newline at end of file
+run-20240211_102016-lsbwqn83
\ No newline at end of file
